### PR TITLE
#164679240 Database setup

### PR DESCRIPTION
**Pull request by rukundoeric**
### What does this do?
* Setup database connection

#### What are the relevant pivotal tracker stories?
* #164679240

### CODE REVIEW

File: server/helpers/Helper.js [hunk 1]
```diff
@@ -4,12 +4,6 @@ class Helper {
    constructor(){
        dotenv.config();
        this.bcrpt = bcrpt;
-       this.isStrongPassword = password => {
```
Comment (ndero) [inline server/helpers/Helper.js:7]: Why are you removing the strongPassword validation?

File: server/db/connection/con_file.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+            host: '127.0.0.1',
+            user: process.env.TODO_DB_USER,
+            password: process.env.TODO_DB_PW,
+            database: 'epicmail'
+        }
+        this.production = {
+            connection: process.env.DATABASE_URL
+        }
+        this.getConnectionConfig = () => {
+            return this.environment == 'development' ? this.development : this.production;
```
Comment (ndero) [inline server/db/connection/con_file.js:17]: Why `==` for comparison?

File: server/db/connection/connection.js [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+import dotenv from 'dotenv';
+import con from './con_file';
+dotenv.config();
+const config = con.getConnectionConfig();
+class Connection {
+	constructor(){
+		this.getPoolConnection = () => {
+			return new Pool(config);
+		}
+	}
```
Comment (ndero) [inline server/db/connection/connection.js:11]: The indentation (line 7 to 10) is more than the usual 2 spaces for each level

**Review by ndero — CHANGES_REQUESTED** [review]
Resolve the comments on this PR.

File: server/helpers/Helper.js (hunk 1 above)
Comment (rukundoeric) [reply server/helpers/Helper.js:7]: because joe test it too, so it was somehow duplication

**Review by ndero — APPROVED** [review]

**Pull request merged**